java/gazelle/configure.go

```diff
@@ -102,6 +107,8 @@ func (jc *Configurer) Configure(c *config.Config, rel string, f *rule.File) {
 				cfg.SetTestMode(d.Value)
 			case javaconfig.ExcludeArtifact:
 				cfg.AddExcludedArtifact(d.Value)
+			case javaconfig.TestFilePatterns:
+				cfg.SetTestFilePatterns(d.Value)
```

Comment:
Can we keep this switch case sorted? And with the same whitespace line after the cases.

Reply:
Done